### DISCUSSION TITLE
Sort posts on the same day

### DISCUSSION THREAD

**Discussion by kaitoukid-1412** [facebook/docusaurus#8073] [Q&A] [answered]
Hi,

![image](https://user-images.githubusercontent.com/15698245/189345023-7116ec3f-48df-4015-9d1f-3cd87322cc4d.png)
![image](https://user-images.githubusercontent.com/15698245/189345609-e3297f53-9270-49bb-ac58-4216797c819b.png)
![image](https://user-images.githubusercontent.com/15698245/189345492-e7392b1a-7aba-404b-922a-565622f5ae7a.png)
![image](https://user-images.githubusercontent.com/15698245/189346012-c8542127-6c2f-4bc0-a4b0-ecf6f958072a.png)

☝ This list of posts is not sorted in ascending order

```
example 1
example 10 <- here !?
example 2
example 3
//...
example 9
```

- Docusaurus v2.1.0

**Comment by Josh-Cena** [comment, marked as answer]
You can see https://docusaurus.io/docs/blog#blog-post-date:

> Since the front matter follows YAML syntax where the datetime notation is supported, you can use front matter if you need more fine-grained publish dates. For example, if you have multiple posts published on the same day, you can order them according to the time of the day:
